### PR TITLE
docs: Add JumpCloud SSO + sync docs

### LINKED ISSUE #4950: Before shipping JumpCloud integration
- [ ] Test further on staging
- [x] #4973
- [x] Write docs for JumpCloud (be sure to mention group filter matching)

### PR DESCRIPTION
**Pull request by jamilbk**
Adds a step-by-step guide for setting up the new JumpCloud connector and optionally directory sync.

Fixes #4950

### CODE REVIEW

**Comment by jamilbk** [comment]
@bmanifold Ready for review.

File: website/src/app/kb/authenticate/jumpcloud/readme.mdx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+  href="/images/kb/authenticate/jumpcloud/3-select-options.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/jumpcloud/3-select-options.png"
+    alt="Select options"
+    width={1200}
+    height={1200}
+  />
+</Link>
```
Comment (bmanifold) [inline website/src/app/kb/authenticate/jumpcloud/readme.mdx:100]: Looks like this image is repeated, was that intentional?

Comment (jamilbk) [reply website/src/app/kb/authenticate/jumpcloud/readme.mdx:100]: Good catch, this is now fixed.

**Review by bmanifold — APPROVED** [review]
LGTM

**Pull request merged**